### PR TITLE
BLB/BLC update 18-07 #3

### PR DESCRIPTION
**Pull request by Fulgur14**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/bloodroot_apothecary.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Name:Bloodroot Apothecary
+ManaCost:2 G
+Types:Creature Squirrel Druid
+PT:3/3
+K:Toxic:2
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigToken | TriggerDescription$ WWhen CARDNAME enters, you and target opponent each create a Treasure token.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/bloodroot_apothecary.txt:6]: typo

File: forge-gui/res/cardsfolder/upcoming/calamity_of_cinders.txt [hunk 2]
```diff
@@ -0,0 +1,6 @@
+Name:Calamity of Cinders
+ManaCost:5 R R
+Types:Sorcery
+K:Convoke
+A:SP$ DamageAll | ValidCards$ Creature.untapped | NumDmg$ 6 | ValidDescription$ each untapped creature. | SpellDescription$ CARDNAME deals 6 damage to each untapped creature.
+Oracle:Convoke (Your creatures can help cast this spell. Each creature you tap while  casting this spell pays for {1} or one mana of that creature's color.)\nCalamity of Cinders deals 6 damage to each untapped creature.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/calamity_of_cinders.txt:6]: spacing

File: forge-gui/res/cardsfolder/upcoming/fortune_tellers_talent.txt [hunk 3]
```diff
@@ -0,0 +1,10 @@
+Name:Fortune Teller's Talent
+ManaCost:U
+Types:Enchantment Class
+S:Mode$ Continuous | Affected$ Card.TopLibrary+YouCtrl | AffectedZone$ Library | MayLookAt$ You | Description$ You may look at the top card of your library any time.
+K:Class:2:3 U:AddStaticAbility$ SFutureSight
+SVar:SFutureSight:Mode$ Continuous | Affected$ Card.TopLibrary+YouCtrl | CheckSVar$ X | SVarCompare$ GE1 | AffectedZone$ Library | MayPlay$ True | Description$ As long as you've cast a spell this turn, you may play cards from the top of your library.
+K:Class:3:2 U:AddStaticAbility$ SReduceCost
+SVar:SReduceCost:Mode$ ReduceCost | ValidCard$ Card.wasNotCastFromYourHand | Type$ Spell | Activator$ You | Amount$ 2 | Description$ Spells you cast from anywhere other then your hand cost {2} less to cast.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/fortune_tellers_talent.txt:8]: "then"

File: forge-gui/res/cardsfolder/upcoming/insatiable_frugivore.txt [hunk 4]
```diff
@@ -0,0 +1,13 @@
+Name:Insatiable Frugivore
+ManaCost:3 B
+Types:Creature Rat Berserker
+PT:2/4
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigToken | TriggerDescription$ When CARDNAME enters, create a Food token, then you may exile three cards from your graveyard. If you do, repeat this process.
+SVar:TrigToken:DB$ Token | TokenAmount$ 1 | TokenOwner$ You | TokenAmount$ 1 | TokenScript$ c_a_food_sac | SubAbility$ DBRepeat
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/insatiable_frugivore.txt:6]: duplicated params

File: forge-gui/res/cardsfolder/upcoming/jacked_rabbit.txt [hunk 5]
```diff
@@ -0,0 +1,11 @@
+Name:Jacked Rabbit
+ManaCost:X 1 W
+Types:Creature Rabbit Warrior
+PT:1/2
+K:Ravenous
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigToken | TriggerZones$ Battlefield | TriggerDescription$ Whenever CARDNAME enters the battlefield or attacks, create a number of 1/1 colorless Gnome artifact creature tokens equal to its power.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/jacked_rabbit.txt:6]: wrong description...

File: forge-gui/res/cardsfolder/upcoming/insatiable_frugivore.txt (hunk 4 above)
[the remark this answers is not in the document]
Comment (Fulgur14) [reply forge-gui/res/cardsfolder/upcoming/insatiable_frugivore.txt:6]: I tested it, though I haven't tried to exile the cards more than once... But it worked.

File: forge-gui/res/cardsfolder/upcoming/tempt_with_bunnies.txt [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+ManaCost:2 W
+Types:Sorcery
+A:SP$ Draw | SubAbility$ DBToken1 | SpellDescription$ Tempting offer — Draw a card and create a 1/1 white Rabbit creature token. Then each opponent may draw a card and create a 1/1 white Rabbit creature token. For each opponent who does, you draw a card and you create a 1/1 white Rabbit creature token.
+SVar:DBToken1:DB$ Token | TokenScript$ w_1_1_rabbit | TokenAmount$ 1 | TokenOwner$ You | SubAbility$ DBRepeat
+SVar:DBRepeat:DB$ RepeatEach | RepeatSubAbility$ DBDraw1 | RepeatPlayers$ Player.Opponent | SubAbility$ DBDraw2 | ChangeZoneTable$ True | RepeatOptionalForEachPlayer$ True | RepeatOptionalMessage$ Do you want to draw a card and create a 1/1 white Rabbit creature token?
+SVar:DBDraw1:DB$ Draw | Defined$ Player.IsRemembered | SubAbility$ DBToken2
+SVar:DBToken2:DB$ Token | TokenScript$ w_1_1_rabbit | TokenAmount$ 1 | TokenOwner$ Player.IsRemembered | SubAbility$ DBCount
+SVar:DBCount:DB$ StoreSVar | SVar$ X | Type$ CountSVar | Expression$ X/Plus.1
+SVar:DBDraw2:DB$ Draw | NumCards$ X | SubAbility$ DBToken3
+SVar:DBToken1:DB$ Token | TokenScript$ w_1_1_rabbit | TokenAmount$ X | TokenOwner$ You | SubAbility$ DBReset
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/tempt_with_bunnies.txt:11]: wrong name

File: forge-gui/res/cardsfolder/upcoming/insatiable_frugivore.txt (hunk 4 above)
[the remark this answers is not in the document]
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/insatiable_frugivore.txt]: the problem is if you don't do it and the trigger gets copied `CheckNotPaid` will already be 0 when it resolves the second time

[the remark this answers is not in the document]
Comment (Fulgur14) [reply forge-gui/res/cardsfolder/upcoming/insatiable_frugivore.txt]: So what should I do?

[the remark this answers is not in the document]
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/insatiable_frugivore.txt]: just reset it to 1 as the first thing

[the remark this answers is not in the document]
Comment (Fulgur14) [reply forge-gui/res/cardsfolder/upcoming/insatiable_frugivore.txt]: Like this?

**Review by tool4ever — APPROVED** [review]

**Pull request merged**